### PR TITLE
Send and receive data from an API

### PR DESCRIPTION
**Pull request by harshitajain06**
Added following functionalities:
- Sent and received data from an API.
- Used API documentation.
- Understood and used JSON.
- Made JavaScript code asynchronous.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,18 +6,27 @@ import TitleAuthor from './modules/book.js';
...
+      score: author.value,
+    }),
+    headers: {
+      'Content-Type': 'application/json;charset=UTF-8',
+    },
   });
+  window.location.reload();
 });
+
+refresh.addEventListener('click', () => {
```
Comment (tufoinnkuo10) [inline src/index.js:30]: - [ ] nice work so far.  when the submit button is clicked, the score list changes. According to the project guidelines, When the Refresh button is clicked, it should hit the GET API endpoint before  the scores list  can change. please kindly fix that.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @harshitajain06 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by ahmetbozaci** [comment]
### Hello @harshitajain06 
### _This is an additional review as a result of the second opinion request._ 



- There is no requirement for submit button cannot refresh the list.
- Submit and refresh buttons working as expected.

___
_**Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away**_




### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by ichala — APPROVED** [review]
Hi @harshitajain06   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**